docs/changelog/87361.yaml

```
@@ -3,4 +3,17 @@ summary: "Implement per-transform num_failure_retries setting"
 area: Transform
 type: enhancement
 issues: []
+highlight:
+  title: Infinite and adaptive retries for Transforms
+  body: |-
+   Infinite and adaptive retires – available in 8.4 – makes it possible for 
+   transforms to recover after a failure without any user intervention. Retries 
+   can be configured per transform. The transform retries become less frequent 
+   progressively. The interval between retries doubles after reaching a one-hour 
+   threshold. This is because the possibility that retries solve the problem is 
+   less likely after each failed retry.
+   
+   In the Transforms page in Stack Management in Kibana, the number of retries 
```

Is it possible to use bold and shared attributes in these summaries? If so:
```suggestion
   In the *Transforms* page in *{stack-manage-app}* in {kib}, the number of retries 
```